docs/creating-a-package.md

```diff
@@ -1,6 +1,6 @@
 # Creating Packages
 
-Packages are at the core of Atom. Nearly everything outside of the main editor
+Packages are at the core of Atom. You can write a package in Javascript or CoffeeScript. Nearly everything outside of the main editor
```

Comment:
I like that suggestion, @50Wliu. :+1: 

@JoeArizona Also, what do you think about moving this sentence to one of the next paragraphs? I think it might fit a bit better there -- it breaks the flow of this paragraph a bit, and it's not the most important thing to mention at the start of the guide, I think.